addon/components/hyper-table/filters-renderers/text.js

```diff
@@ -20,11 +20,13 @@ export default Component.extend(FiltersRendererMixin, {
   }),
 
   _addSearchFilter() {
-    this.column.set(
-      'filters',
-      isEmpty(this._searchQuery) ? [] : [{ key: 'value', value: this._searchQuery }]
-    );
-    this.manager.hooks.onColumnsChange('columns:change');
+    if(this._searchQuery !== null) {
```

Comment:
```js
if(!this._searchQuery) {
    return
}```

Reply:
Check should be only when null, empty strings are okay, otherwise it doesn't update when search term is removed